test/widget_test.dart

```diff
@@ -12,7 +12,5 @@ void main() {
   testWidgets('show WH org name on home page', (tester) async {
     // dont use _initApp as for this we *dont* want the future to complete yet
     await tester.pumpWidget(MyApp());
```

Comment:
Just for posterity: I told @alessandrofavero  he could disable the test and I would fix up the test to work with his changes in subsequenet PR